### PR TITLE
Add VM settings

### PR DESCRIPTION
**Pull request by ddalu5**
Add VM settings model

### CODE REVIEW

File: app/manager/models.py [hunk 1]
```diff
@@ -108,19 +108,26 @@ def __str__(self):
         return self.name
 
 
+class VMSettings(models.Model):
```
Comment (EricDele) [inline app/manager/models.py]: I think hardware is better for the name of this class as this is not only a vm but a bare metal server

**Comment by ddalu5** [comment]
Ok, I'll do it tonight, check if you see anything else please ;)

**Comment by EricDele** [comment]
Oh, I missed the generation filed that have to be in the hardware class, sorry.

**Comment by ddalu5** [comment]
No worries, thanks a lot ;)

File: app/manager/models.py (hunk 1 above)
Comment (ddalu5) [reply app/manager/models.py]: Done :D

**Pull request merged**